modin/pandas/test/test_groupby.py

```diff
@@ -1466,6 +1466,62 @@ def test_agg_exceptions(operation):
     eval_aggregation(*create_test_dfs(data), operation=operation)
 
 
+@pytest.mark.parametrize("numeric_only", [True, False])
+def test_mean_agg_different_types(numeric_only):
```

Comment:
let's also add a `mean` test to the [`test_groupby_multiindex`](https://github.com/modin-project/modin/blob/41213581b974760928ed36f05c479937dcdfea55/modin/pandas/test/test_groupby.py#L1266) to check whether our intermediate columns renaming works with them